.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -742,18 +757,39 @@ private DateTimeResolutionResult ParseDayToDay(string text, DateObject reference
                     beginDay = this.config.DayOfMonth[dayFrom];
                     endDay = this.config.DayOfMonth[dayTo];
                 }
+                else if (match.Groups["half"].Success)
```

Comment:
Minor: Named group names should be constants.

Reply:
replaced all names